src/libraries/Native/native-binplace.proj

```diff
@@ -11,16 +11,18 @@
   <!-- Ordering matters! Overriding GetBinPlaceItems and Build targets after the Sdk import. -->
   <Target Name="GetBinPlaceItems">
     <ItemGroup>
-      <BinPlaceItem Include="$(NativeBinDir)*.dll" />
-      <BinPlaceItem Include="$(NativeBinDir)*.pdb" />
-      <BinPlaceItem Include="$(NativeBinDir)*.lib" />
-      <BinPlaceItem Include="$(NativeBinDir)*.a" />
-      <BinPlaceItem Include="$(NativeBinDir)*.bc" />
-      <BinPlaceItem Include="$(NativeBinDir)*.so" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dbg" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dylib" />
-      <BinPlaceItem Include="$(NativeBinDir)*.dwarf" />
-      <FileWrites Include="@(BinPlaceItem)" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dll" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.pdb" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.lib" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.a" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.bc" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.so" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dbg" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dylib" />
+      <NativeBinPlaceItems Include="$(NativeBinDir)*.dwarf" />
+
+      <BinPlaceItem Condition="'$(BinPlaceTestRuntimePack)' != 'true'" Include="@(NativeBinPlaceItems)" />
```

Comment:
Why do we have the `Condition` here vs. on the containing `Target`?

Reply:
I think that just needs cleaned up.

Reply:
Because the target should always run. `BinPlaceItem` is going to be used for the regular bin placing, `NativeBinPlaceItems` only when binplacing the test runtime pack. 

The reason why we can't do it the other way around is because `BinPlaceItem` is the default item group we use to binplace.

Reply:
@safern explained our bin placing logic offline so I get why my suggestion won't work. 

I'm now left with hopefully a much simpler question: why `NativeBinPlaceItems` instead of `NativeBinPlaceItem`? Basically don't pluralize since `BinPlaceItem` doesn't pluralize :smile:

Reply:
Yeah, that makes sense. Let's be consistent and have both in singular :smile:

Reply:
Done, changed to singular